synapse/tests/test_model_files.py

```diff
@@ -628,3 +631,6 @@ async def test_model_file_lnk(self):
             self.eq(node.get('target:created'), time)
             self.eq(node.get('target:accessed'), time)
             self.eq(node.get('target:written'), time)
+
+            self.eq(node.get('driveserial'), 0x6af54670)
+            self.eq(node.get('machineid'), 'stellarcollapse')
```

Comment:
A 1-line test to confirm the relationship from the `file:mime:lnk -> it:hostname` here would be a good idea. Something like:
```
self.len(1, await core.nodes('file:mime:lnk -> it:hostname'))
```